packages/cli/src/lib/autorun/autorun-command.unit.test.ts

```diff
@@ -1,18 +1,23 @@
-import { bundleRequire } from 'bundle-require';
 import { vol } from 'memfs';
 import { describe, expect, it, vi } from 'vitest';
 import { PortalUploadArgs, uploadToPortal } from '@code-pushup/portal-client';
-import { collectAndPersistReports } from '@code-pushup/core';
+import {
+  collectAndPersistReports,
+  readCodePushupConfig,
+} from '@code-pushup/core';
 import { MEMFS_VOLUME, MINIMAL_REPORT_MOCK } from '@code-pushup/testing-utils';
 import { DEFAULT_CLI_CONFIGURATION } from '../../../mocks/constants';
 import { yargsCli } from '../yargs-cli';
 import { yargsAutorunCommandObject } from './autorun-command';
 
 vi.mock('@code-pushup/core', async () => {
+  const { CORE_CONFIG_MOCK }: typeof import('@code-pushup/testing-utils') =
+    await vi.importActual('@code-pushup/testing-utils');
   const core = await vi.importActual('@code-pushup/core');
   return {
     ...(core as object),
```

Comment:
```suggestion
  const core: typeof import('@code-pushup/core') =
    await vi.importActual('@code-pushup/core');
  return {
    ...core,
```

(Applies to `collect-command.unit.test.ts` and `upload-command.unit.test.ts`, too.)

Reply:
That's how it's fixed? Brilliant, I remember we ran into this in one of the previous PRs but dismissed it. Glad it has a proper solution 😁

Reply:
Yeah, the `const module: typeof import('<path>') = await vi.importActual('<path>')` pattern can be applied in general. The type has to annotated explicitly because I don't believe it's possible for `vi.importActual` to infer it from the path, only built-in language constructs can do that (i.e. static `import` and dynamic `import()`).